modules/installation-vsphere-regions-zones.adoc

```diff
@@ -11,7 +11,7 @@
 [id="installation-vsphere-regions-zones_{context}"]
 = VMware vSphere region and zone enablement
 
-You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage that can cause your cluster to fail.
+You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage that can cause your cluster to fail. To enable a regional and zone enablement environment, you must define multiple failure domains for your {product-title} cluster.
```

Comment:
```suggestion
You can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. Each datacenter can run multiple clusters. This configuration reduces the risk of a hardware failure or network outage that can cause your cluster to fail. To enable a regions and zones, you must define multiple failure domains for your {product-title} cluster.
```

perhaps reworded like above?

Reply:
Hi @rvanderp3 . Yes, that would be better. Updated.